Tools/autotest/common.py

```diff
@@ -5698,7 +5705,9 @@ def poll_home_position(self, quiet=True, timeout=30):
                 continue
             m = self.mav.messages.get("HOME_POSITION", None)
             if m is None:
-                continue
+                m = self.mav.recv_match(type='HOME_POSITION', blocking=True, timeout=5)
```

Comment:
This function looks to have been carefully constructed NOT to use `recv_match`.

Was the current implementation causing problems?